src/libs/pr.ts

```diff
@@ -12,10 +12,14 @@ export async function handlePullRequestMessage(
     githubToken,
     command,
     stackName,
+    projectName,
     options: { editCommentOnPr },
   } = config;
 
-  const heading = `#### :tropical_drink: \`${command}\` on ${stackName}`;
+  let heading = `#### :tropical_drink: \`${command}\` on ${stackName}`;
+  if (projectName) {
+    heading = `${heading} for project ${projectName}`
+  }
```

Comment:
LGTM. 

Nit: Assume `projectName` is set. Do you think the message would be clear enough if written as 

```markdown
#### :tropical_drink: \`${command}\` on ${projectName}/${stackName}`
```
...or do you think having `for project ${projectName}` is necessary for clarity?

I'm leaning towards "not clear enough" but WDYT?

Reply:
I think projectName/stackName is enough IMO, if the name of the project is something indicative. Not sure what's the "norm" for Pulumi project names, but for the ones I use, the simpler output is better 😄